test/udp_backend_test.rb

```diff
@@ -115,6 +115,13 @@ def test_warn_when_using_tags_and_not_on_datadog
     StatsD.increment('fooc', tags: ['ignored'])
   end
 
+  def test_support_tags_whitelist_on_datadog
```

Comment:
Can you add a test which rejects a tag that is not in the whitelist as well?

Reply:
I believe this test does that?  I have changed it to host and hostname to test the possible prefix collision there